index.html

```diff
@@ -0,0 +1,160 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/index.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css">
+
+
+    <!-- Fonts -->
+    <link href="https://fonts.googleapis.com/css?family=Lato&display=swap" rel="stylesheet">
+
+    <!-- Font Awesome -->
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.7.2/css/all.css"
+        integrity="sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr" crossorigin="anonymous">
+    <title>Document</title>
+</head>
+
+<body>
+
+
+    <header>
+        <div class="topnav">
+            <nav>
+                <li><a href=""><i class="fa-brands fa-instagram-square"></i></a></li>
+                <li><a href=""> <i class="fa-brands fa-twitter"></i></a></li>
+                <li><a href=""><i class="fa-brands fa-facebook"></i></a></li>
+            </nav>
+        </div>
+        <div class="mainnav">
+            <h1 class="main-h1"> The chance</h1>
+            <nav>
+                <li><a href="/about.html"> About</a> </li>
+                <li> Contact</li>
+                <li> Sponsor</li>
+                <li>News</li>
+                <button class="main-button"> Buy Ticket now</button>
+            </nav>
+        </div>
+
+
+        <button type="button" class="ham">
+            <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+                <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M4 6h16M4 12h16M4 18h16" />
+            </svg>
+        </button>
```

Comment:
- [ ] Please make sure that your hamburger button is working as when I click on it it doesn't show the mobile menu.

Reply:
- [ ] Great job by implementing the changes requested by the previous reviewer about the hamburger menu. but, it seems that after implementing those changes, your design changes a little bit.

| Your Design(mobile menu) | Original Design |
| ----------------------------- | ----------------- |
| ![image](https://user-images.githubusercontent.com/71874425/169600009-f569f16e-9f34-4d3d-8732-1dc4e571d69f.png) | ![image](https://user-images.githubusercontent.com/71874425/169600285-b0c08c5a-f8ad-46ad-8843-a27c8af6b35a.png)|
- Kindly, make sure that your project look exactly like the original design provided.
- The content should be centered as on the original design. Let's rock and fix this issue.